docs/src/main/asciidoc/security.adoc

```diff
@@ -55,35 +55,39 @@ Please see xref:security-built-in-authentication.adoc#mutual-tls[Mutual TLS Auth
 
 === OpenID Connect
 
-`quarkus-oidc` extension provides a reactive, interoperable, multi-tenant enabled OpenID Connect adapter which supports `Bearer Token` and `Authorization Code Flow` authentication mechanisms.
+OpenID Connect (OIDC) is an open authentication protocol that enables client applications to verify the identity of a user when that user tries to connect to a secure server, such as HTTPS.
+OIDC uses authorization and authentication mechanisms on OAuth 2.0 compliant authorization servers and builds identities that uniquely identify users.
```

Comment:
@sheilamjones Lets remove this sentence since the above suggestion should be sufficient